op-node/eth/types.go

```diff
@@ -236,6 +238,33 @@ type PayloadAttributes struct {
 	GasLimit *Uint64Quantity `json:"gasLimit,omitempty"`
 }
 
+type L2BatchJustification struct {
+	PrevBatchLastBlock espresso.Header `json:"prevBatchLastBlock"`
+	FirstBlock espresso.Header `json:"firstBlock"`
+
+	// The block number of `FirstBlock`. This allows us to authenticate the `FirstBlock` header by
```

Comment:
Can we store the block number in the header? Or is this not possible right now because `next_block` is stateless?